database_schemas/cassandra/benchflow.cql

```diff
@@ -262,33 +263,144 @@ CREATE TABLE faban_driver_delay_times (
   passed boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), host, driver_name, op_name)
+  PRIMARY KEY (experiment_id, trial_id, host, driver_name, op_name)
 );
 
 CREATE TABLE faban_driver_custom_stats (
   host text,
   driver_name text,
   stat_name text,
   description text,
+  target text,
   result double,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), host, driver_name, stat_name, description)
+  PRIMARY KEY (experiment_id, trial_id, host, driver_name, stat_name, description)
 );
 
 CREATE TABLE faban_details (
   host text,
   section text,
   op_name text,
   time text,
+  time_unit text,
   value double,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((trial_id, experiment_id), host, section, op_name, time)
+  PRIMARY KEY (experiment_id, trial_id, host, section, op_name, time)
 );
 
 -- Experiment metrics
+CREATE TABLE exp_faban_total_ops (
+  experiment_id text,
+  faban_driver_host text,
+  faban_driver_name text,
+  faban_total_ops_mean double,
+  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name)
+);
+
+CREATE TABLE exp_faban_ops (
+  experiment_id text,
+  faban_driver_host text,
+  faban_driver_name text,
+  faban_op_name text,
+  faban_successes_mean double,
+  faban_successes_min double,
+  faban_successes_max double,
+  faban_successes_q1 double,
+  faban_successes_q2 double,
+  faban_successes_q3 double,
+  faban_successes_p90 double,
+  faban_successes_p95 double,
+  faban_successes_p98 double,
+  faban_successes_percentiles list<double>,
+  faban_successes_ci095_min double,
+  faban_successes_ci095_max double,
+  faban_successes_me double,
+  faban_successes_mode list<double>,
+  faban_successes_mode_freq bigint,
+  faban_successes_sd double,
+  faban_failures_mean double,
+  faban_failures_min double,
+  faban_failures_max double,
+  faban_failures_q1 double,
+  faban_failures_q2 double,
+  faban_failures_q3 double,
+  faban_failures_p90 double,
+  faban_failures_p95 double,
+  faban_failures_p98 double,
+  faban_failures_percentiles list<double>,
+  faban_failures_ci095_min double,
+  faban_failures_ci095_max double,
+  faban_failures_me double,
+  faban_failures_mode list<double>,
+  faban_failures_mode_freq bigint,
+  faban_failures_sd double,
+  faban_mix_mean double,
+  faban_mix_min double,
+  faban_mix_max double,
+  faban_mix_q1 double,
+  faban_mix_q2 double,
+  faban_mix_q3 double,
+  faban_mix_p90 double,
+  faban_mix_p95 double,
+  faban_mix_p98 double,
+  faban_mix_percentiles list<double>,
+  faban_mix_ci095_min double,
+  faban_mix_ci095_max double,
+  faban_mix_me double,
+  faban_mix_mode list<double>,
+  faban_mix_mode_freq bigint,
+  faban_mix_sd double,
+  faban_delay_times_weighted_avg double,
+  faban_delay_times_min double,
+  faban_delay_times_max double,
+  PRIMARY KEY (experiment_id, faban_driver_host, faban_driver_name, faban_op_name)
+);
+
+CREATE TABLE exp_faban_ops_response_times (
```

Comment:
They are already stats, so the min/max is the min/max of the recorded stat, so if we have collected these data:
'
experiment_id   | trial_id | host      | driver_name | op_name      | stat_name | passed | stat_value
-----------------+----------+-----------+-------------+--------------+-----------+--------+------------
 camundaZZZZZZMV |        C | aggregate |   WebDriver | MyOperation1 |       avg |   True |      0.001
 camundaZZZZZZMV |        C | aggregate |   WebDriver | MyOperation1 |       max |   True |      0.001
 camundaZZZZZZMV |        C | aggregate |   WebDriver | MyOperation1 |     p90th |   True |        100
 camundaZZZZZZMV |        C | aggregate |   WebDriver | MyOperation1 |     p99th |   True |        100
 camundaZZZZZZMV |        C | aggregate |   WebDriver | MyOperation1 |        sd |   True |          0
'
then we will have the min/max of all those, which is all we can do, since they can be dynamic.